pkg/kube/controllers/quarksstatefulset/quarksstatefulset_reconciler.go

```diff
@@ -228,44 +226,31 @@ func (r *ReconcileQuarksStatefulSet) createStatefulSet(ctx context.Context, qSta
 		return errors.Wrapf(err, "could not set owner for StatefulSet '%s' to QuarksStatefulSet '%s' in namespace '%s'", statefulSet.Name, qStatefulSet.Name, qStatefulSet.Namespace)
 	}
 
-	// Create the StatefulSet
-	if err := r.client.Create(ctx, statefulSet); err != nil {
-		return errors.Wrapf(err, "could not create StatefulSet '%s' for QuarksStatefulSet '%s' in namespace '%s'", statefulSet.Name, qStatefulSet.Name, qStatefulSet.Namespace)
+	statefulSetMutateFn := func(sfs *v1beta2.StatefulSet) controllerutil.MutateFn {
```

Comment:
We had a common package to handle mutates

Reply:
https://github.com/cloudfoundry-incubator/cf-operator/blob/master/pkg/kube/util/mutate/mutate.go

Reply:
I moved it